vtrace/platforms/win32.py

```diff
@@ -1171,6 +1198,8 @@ class TI_FINDCHILDREN_PARAMS(Structure):
     return tif
 
 def raiseWin32Error(name):
+    mesg = "Win32 Error %s failed: %s" % (name, kernel32.GetLastError())
+    logger.warning(mesg)
     raise vtrace.PlatformException("Win32 Error %s failed: %s" % (name,kernel32.GetLastError()))
```

Comment:
i think you intended to merge these together more.  you shouldn't need to call kernel32.GetLastError() twice.